modules/microvm/virtualization/microvm/appvm.nix

```diff
@@ -42,10 +47,18 @@
             if vm.borderColor != null
             then "--border \"${vm.borderColor}\""
             else "";
-          runWaypipe = pkgs.writeScriptBin "run-waypipe" ''
-            #!${pkgs.runtimeShell} -e
-            ${pkgs.waypipe}/bin/waypipe --vsock -s ${toString configHost.ghaf.virtualization.microvm.guivm.waypipePort} ${waypipeBorder} server "$@"
-          '';
+          runWaypipe =
+            if shmConfig.display
+            then
+              pkgs.writeScriptBin "run-waypipe" ''
```

Comment:
Is it possible to have `if, else` in script rather creating than creating different scripts?